src/org/loklak/harvester/strategy/KaizenHarvester.java

```diff
@@ -94,6 +109,17 @@ private void grabInformation(Timeline timeline) {
             if (!place.isEmpty())
                addQuery("near:\"" + message.getPlaceName() + "\" within:" + PLACE_RADIUS + "mi");
         }
+
+        if (query != null && oldestTweetDate != null) {
+            String oldestTweetDateStr = dateToString.format(oldestTweetDate);
+            int startIndex = query.indexOf("until:");
```

Comment:
Try out these searches on Twitter - 
https://twitter.com/search?q=since%3A2017-04-21%20el%20clasico&src=typd
https://twitter.com/search?q=until%3A2017-04-21%20el%20clasico&src=typd